packages/good-design/src/apps/MBP/MPB.tsx

```diff
@@ -0,0 +1,861 @@
+  const [newProvider, setProvider] = useState<JsonRpcProvider | undefined>(
+    new ethers.providers.JsonRpcProvider("https://rpc.ankr.com/eth", "any")
+  );
+
+  const { showNotification, notification, onClose } = useNotification();
+  const [visibility, setVisibility] = useState("none");
+  const [amount, setAmount] = useState("");
+  const [label, setLabel] = useState("Connect Wallet");
+  const [signer, setSigner] = useState([]);
+  const [contractAddress, setContractAddress] = useState("0x67C5870b4A41D4Ebef24d2456547A03F1f3e094B");
+  const [srcChainId, setSrcChainId] = useState(1);
+  const [dstChainId, setDstChainId] = useState(1);
+  const [userAddress, setUserAddress] = useState("");
+  const [bridgeprovider, setBridgeProvider] = useState("Axelar");
+  const [srcNetworkImg, setSrcNetworkImg] = useState(ethereum);
+  const [srcNetwork, setSrcNetwork] = useState("Ethereum");
+  const [dstNetwork, setDstNetwork] = useState("Celo");
+  const [srcNetworkCounter, setSrcNetworkCounter] = useState(0);
+  const [feeData, setFeeData] = useState({});
+  const [fee, setFee] = useState("/");
+  const [txHash, setTxHash] = useState("");
+  const [expLnk, setExpLnk] = useState("");
+  const [transition, setTransition] = useState("none");
+  const [transform, setTransform] = useState("none");
+  const [angel, setAngel] = useState(-360);
+  const balanceFormatted = useG$Amount(BigNumber.from(balance), "G$", 1);
+  const [borderBottom, setBorderBottom] = useState({
+    ethereum: "3px",
+    celo: "3px",
+    fuse: "3px",
+    axelar: "3px",
+    layerzero: "3px"
+  });
+  const [top, setTop] = useState({
+    ethereum: "0px",
+    celo: "0px",
+    fuse: "0px",
+    axelar: "0px",
+    layerzero: "0px"
+  });
+  const [buttonColors, setButtonColors] = useState({
+    ethereum: {
+      backgroundColor: "#ededed",
+      borderColor: "#c9c9c9"
+    },
+    celo: {
+      backgroundColor: "#ffffff",
+      borderColor: "rgb(0, 174, 255)"
+    },
+    fuse: {
+      backgroundColor: "#ffffff",
+      borderColor: "rgb(0, 174, 255)"
+    },
+    axelar: {
+      backgroundColor: "#ffffff",
+      borderColor: "rgb(0, 174, 255)"
+    },
+    layerzero: {
+      backgroundColor: "#ffffff",
+      borderColor: "rgb(0, 174, 255)"
+    }
+  });
+  const [inputColor, setInputColor] = useState({
+    address: "black",
+    amount: "black"
+  });
+
+  const handleSelect = buttonName => {
+    const nets = ["ethereum", "celo", "fuse"];
+    const providers = ["axelar", "layerzero"];
+    if (buttonName !== srcNetwork.toLowerCase()) {
+      setBorderBottom(prevBorderWidth => ({
+        ...prevBorderWidth,
+        [buttonName]: "1px"
+      }));
+      setTop(prevTop => ({
+        ...prevTop,
+        [buttonName]: "3px"
+      }));
+      if (nets.includes(buttonName)) {
+        setDstNetwork(buttonName.charAt(0).toUpperCase() + buttonName.slice(1));
+        nets.map((net, index) => {
+          if (buttonColors[net].backgroundColor !== "#ededed") {
```

Comment:
remove duplication.
the color update is identical for both, could be done with a helper method